### PR TITLE
Add mapping for PV max to SwitchoffFile datatype

### LINKED ISSUE #89: SwitchoffFile value '19' seems to be 'PVmax'
From `dump_luxtronik.py`:

Number: 106   Name: ID_WEB_Switchoff_file_Nr0                                    Type: SwitchoffFile        Value: no request
Number: 107   Name: ID_WEB_Switchoff_file_Nr1                                    Type: SwitchoffFile        Value: evu lock
Number: 108   Name: ID_WEB_Switchoff_file_Nr2                                    Type: SwitchoffFile        Value: no request
Number: 109   Name: ID_WEB_Switchoff_file_Nr3                                    Type: SwitchoffFile        Value: no request
Number: 110   Name: ID_WEB_Switchoff_file_Nr4                                    Type: SwitchoffFile        **Value: 19**
Number: 111   Name: ID_WEB_Switchoff_file_Time0                                  Type: Timestamp            Value: 2023-03-11 10:21:44
Number: 112   Name: ID_WEB_Switchoff_file_Time1                                  Type: Timestamp            Value: 2023-03-11 13:18:20
Number: 113   Name: ID_WEB_Switchoff_file_Time2                                  Type: Timestamp            Value: 2023-03-11 13:39:32
Number: 114   Name: ID_WEB_Switchoff_file_Time3                                  Type: Timestamp            Value: 2023-03-11 14:46:23
Number: 115   Name: ID_WEB_Switchoff_file_Time4                                  Type: Timestamp            Value: 2023-03-11 18:26:36

From `datalog`:

11.03.23 19:26 **PVmax**
11.03.23 15:46 keine Anf.
11.03.23 14:39 keine Anf.
11.03.23 14:18 EVU-Sperre
11.03.23 11:21 keine Anf.

Please ignore the timeshift of 1 hour.

Had to modify `dump_luxtronik.py` to get the value
```     
    def from_heatpump(self, value):
        if value in self.codes:
            return self.codes.get(value)
        return None          << supress the assignment of unexpected values
```

### PR DESCRIPTION
**Pull request by Guzz-T**
As demonstrated in #89 the value 19 for the SwitchoffFile datatype represents `PV max`.

This change adds the entry accordingly to make the mapping available for consumers of the datatype module.

Fixes #89.

### CODE REVIEW

**Comment by kbabioch** [comment]
Change itself looks fine, would like to have a more detailed description in the pull request / commit, though.

**Comment by Guzz-T** [comment]
The issue linked above should contain all relevant information. I don't know what else to write about it other than copying the description here :)

File: luxtronik/datatypes.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def options(self):
     def from_heatpump(self, value):
         if value in self.codes:
             return self.codes.get(value)
-        return None
+        return value
```
Comment (kbabioch) [inline luxtronik/datatypes.py:45]: This change is unrelated to this pull request. In case it is needed / wanted, I would suggest to have another pull request, as to not mix up pull requests and discussions.

**Comment by kbabioch** [comment]
I've edited the title and description with some details. You're right of course, that #89 already contains all relevant information. However I would like to have the most basic stuff also added to the commit and merge itself, because this is what will end up in the Git repository. When running `git blame` later on while trying to debug a problem, no one is willing to cross-reference every line of code with some GitHub issues :-).

So, redundancy (e.g. copy & paste) is needed in some places sometimes ;-).

File: luxtronik/datatypes.py (hunk 1 above)
Comment (Guzz-T) [reply luxtronik/datatypes.py]: Done, see #91

Comment (kbabioch) [reply luxtronik/datatypes.py]: Okay, I've rebased the branch in order to not have a "revert" commit that will only confuse history ;-). Thank you for your other issue / PR in #91.

**Review by kbabioch — APPROVED** [review]
